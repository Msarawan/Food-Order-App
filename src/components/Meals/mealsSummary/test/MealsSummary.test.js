import React from "react";
import MealSummary from "../../mealsSummary/MealsSummary"
import { mount } from "enzyme";

describe("Food Order aap testing", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<MealSummary/>);
  });

  test("render the title of meal summary", () => {
    expect(wrapper.find("h2").text()).toContain("Delicious Food, Delivered To You");
  });

  test("render the discription of meal", () => {
    expect(wrapper.find(".summary").text()).toContain("Choose your favorite meal from our broad selection of available meals and enjoy a delicious lunch or dinner at home.");
  });

  test("render the discription", () => {
    expect(wrapper.find(".summary").text()).toContain("All our meals are cooked with high-quality ingredients, just-in-time and of course by experienced chefs!")});

  test('renders a snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
});