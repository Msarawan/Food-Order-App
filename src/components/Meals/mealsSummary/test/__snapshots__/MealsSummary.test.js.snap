// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Food Order aap testing renders a snapshot 1`] = `ReactWrapper {}`;
